### PR TITLE
Remove broken link in tf.keras.callbacks.TensorBoard at docs page

### LINKED ISSUE #48174: https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard contains broken links
## URL(s) with the issue:

https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard

## Description of issue (what needs changing):

The doc for the `embeddings_metadata` argument contains a broken link. It says "See the details about metadata files format.", and the link redirects to https://www.tensorflow.org/how_tos/embedding_viz/#metadata_optional, which is broken.

### LINKED ISSUE #48714: Adding a utility to penalize majority class pixels in the Segmentation tutorial
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/tutorials/images/segmentation

## Description of issue (what needs changing):

It's not really an issue, a suggestion rather. 

### Clear description

Semantic segmentation datasets can be highly imbalanced meaning that particular class pixels can be present more inside images than that of other classes. Since segmentation problems can be treated as per-pixel classification problems we can deal with the imbalance problem by weighing the loss function to account for this. It's a simple and elegant way to deal with this problem. Other solutions include always ensuring that a batch of samples (during training) always contain some proportion (which is prefixed) of positive classes. 

However, TensorFlow does not yet support the `class_weight` argument in `model.fit()` for targets that are 3D (for segmentation problems, we are essentially predicting a map of shape `[batch_size, height, width, nb_channels]`). One way to get around this problem is to use `sample_weight` instead. But then again, it's not very clear as to how to do that properly particularly with `tf.data` pipelines. 

Multiple folks have tried several hacks to get around this problem but it keeps coming back (see [here](https://github.com/keras-team/keras/issues/3653)). Therefore, I think the tutorial under question is a perfect opportunity to demonstrate the use case. 

Cc: @MarkDaoust

### PR DESCRIPTION
**Pull request by alphaX86**
## Description
This PR fixes #48174 

This actually related to docs-error. So, I removed the broken link along with the line. Refer the issue for more details

## Changes made
Removed the link along with the line concerned

## Notes for reviewers
If any changes are to be made/reverted, please let me know! :)

### CODE REVIEW

**Review by mihaimaruseac — COMMENTED** [review]
Is the link moved somewhere else?

**Comment by alphaX86** [comment]
I didn't check but that link throws a 404 error whenever user clicks it

**Comment by alphaX86** [comment]
Any link is there then to replace it? If such, then I'll replace it instead of removing the line

**Comment by alphaX86** [comment]
Everyone I've changed the issue no referenced

**Review by fchollet — APPROVED** [review]
Thanks for the CL!
